### PR TITLE
Improve code highlighting and copy button

### PR DESCRIPTION
**Pull request by martina-if**
- Improved the highlighting by switching from pygments to highlight.js
- Improved the copy button's style
- Added Netlify credit to the land page footer

highlight

### CODE REVIEW

File: site/config.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ baseURL: "https://eksctl.io"
 languageCode: en-us
 title: "eksctl"
 #theme:  "hugo-book-master"
-pygmentsCodeFences: true
+#pygmentsCodeFences: true
 #pygmentsUseClasses: true
-pygmentsStyle: "colorful"
+#pygmentsStyle: "colorful"
```
Comment (cPu1) [inline site/config.yaml]: Should we remove these comments since we're no longer using pygments?

File: site/static/svg/copy.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+    <path fill="#cacaca" d="M6 18H4V6a2 2 0 0 1 2-2h12v2H6z"/>
+    <path fill="#cacaca" d="M28 12v16H12V12h16m0-2H12a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V12a2 2 0 0 0-2-2z"/>
+</svg>
```
Comment (gemagomez) [inline site/static/svg/copy.svg]: Do we need to quote here, even if only as a comment, the source of the button?

File: site/content/_index.md [hunk 3]
```diff
@@ -110,6 +110,8 @@ To learn more about how to create clusters and other features continue reading t
 [![Circle CI](https://circleci.com/gh/weaveworks/eksctl/tree/master.svg?style=shield)](https://circleci.com/gh/weaveworks/eksctl/tree/master) [![Coverage Status](https://coveralls.io/repos/github/weaveworks/eksctl/badge.svg?branch=master)](https://coveralls.io/github/weaveworks/eksctl?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/weaveworks/eksctl)](https://goreportcard.com/report/github.com/weaveworks/eksctl)
 ![Gophers: E, K, S, C, T, & L](introduction/images/eksctl.png)
 
-> **_Logo Credits_**
+> **_Credits_**
 >
 > _Original Gophers drawn by [Ashley McNamara](https://twitter.com/ashleymcnamara), unique E, K, S, C, T & L Gopher identities had been produced with [Gopherize.me](https://gopherize.me/)._
+>
+> Site powered by [Netlify ©](https://netlify.com)
```
Comment (gemagomez) [inline site/content/_index.md]: Looks good!

**Comment by bia** [comment]
This click style should go away on mouse-up and it's not going away for me
Screenshot 2019-07-02 at 15 38 40

also, would be nice to have something like this on hover
Screenshot 2019-07-02 at 15 38 15

**Pull request merged**